### ISSUE TITLE
v5 - showLoading fontSize not working

### ISSUE THREAD

**Issue by wenfangdu** [apache/echarts#13892]
### Version
5.0.0

### Steps to reproduce
```js
      this.chart.showLoading({
        text: 'Loading...',
        color: '#fff',
        textColor: '#fff',
        maskColor: 'rgba(0, 0, 0, 0.5)',
        zlevel: 1,
        fontSize: 30,
      })
```

![image](https://user-images.githubusercontent.com/28700378/103254321-ca818880-49bf-11eb-995f-cf801acac53c.png)

### What is expected?
Font size be set to 30px.

### What is actually happening?
Font size remained default.

**Comment by plainheart** [comment]
Related with #13893.

**Comment by tihuan** [comment]
I think the API is `chart.showLoading(type?: string, opts?: Object)`, so the following works for me!

```ts
chart.showLoading("default", { fontSize: 50 });
```

**Comment by plainheart** [comment]
@tihuan Thanks for your information. This param `type` is optional, so the usage is okay.  `fontSize is not working` is ​a bug and we fixed it in #13972.

**Comment by tihuan** [comment]
Ahh got it, that's great to know! Thanks a lot 😁
